app/src/main/res/values-en/strings.xml

```diff
@@ -0,0 +1,214 @@
+    <string name="exposure_detail_contact_start">Contact</string>
+
+    <string name="exposure_detail_title_none">No exposure was detected</string>
+
+    <string name="exposure_detail_message_none">Corona Alert has not observed any exposure to the coronavirus in the last 14 days.</string>
+    <string name="exposure_detail_disclaimer_none">However, the app cannot identify all possible exposures.</string>
+    <string name="exposure_detail_last_check">Last check %1$s hrs. %2$s</string>
+    <string name="exposure_detail_no_last_check">No checks performed.</string>
+
+    <string name="exposure_detail_instructions_title">Follow these instructions carefully for two weeks:</string>
+    <string name="exposure_detail_instructions_1">Contact your municipal health service as soon as possible and apply for a test if you experience any of these symptoms that are relevant to coronavirus infection:</string>
+    <string name="exposure_detail_instructions_1_sublist">- change in sense of smell or taste \n- difficulty breathing \n- cough, sore throat or runny nose \n- fever \n- headache or muscle aches \n- diarrhea or stomach pain without any other obvious cause.</string>
+    <string name="exposure_detail_instructions_2">Avoid meeting people.</string>
+    <string name="exposure_detail_instructions_3">Practice good hand hygene</string>
+    <string name="exposure_detail_instructions_4">Avoid public places and transport. If you have to stay in public places, observe at least two meters distance from others. If you can not maintain separation, use mouth and nose protection.</string>
+    <string name="exposure_detail_instructions_5">Let others handle your grocery shopping and other errands outside the home, if possible.</string>
+    <string name="exposure_detail_instructions_6">If you cough or sneeze, protect your mouth and nose with a disposable handkerchief. Throw the used handkerchief in the trash and wash your hands with soap and water or use hand sanitizer. If you do not have a handkerchief, cough and sneeze into the sleeve.</string>
+    <string name="exposure_detail_instructions_7">Work remotely, if possible.</string>
+
+    <string name="guide_title">About Corona Alert</string>
+    <string name="guide_p1_text">Corona Alert is an application (app) for which the Institute for Health and Welfare is responsible. The app analyzes whether you may have been exposed to infection. If you have been diagnosed with coronavirus infection, you can report this anonymously via the app. The app effectively protects your identity.</string>
+
+    <string name="guide_code_title">Each device uses a random identification code.</string>
+    <string name="guide_code_text">The app generates an identification code for each device. The code is random and is changed several times per hour so that your identity can not be revealed. The identification code does not contain any information about you or your phone.</string>
```

Comment:
```suggestion
    <string name="guide_code_text">The app generates an identification code for each device. The code is random and changes several times per hour so that your identity cannot be revealed. The identification code does not contain any information about you or your phone.</string>
```